containers/hydra/Dockerfile

```diff
@@ -36,6 +36,7 @@ RUN rm -Rf /root/.cache
 ARG CONTAINER_DEBUG=false
 RUN touch /etc/container-debug-${CONTAINER_DEBUG}
 
+RUN chmod +x /setup/messager.py
```

Comment:
Please move this next to where other files in /setup/ get their executability, or even add this to the very same line (less intermediate docker image layers)